static/navbar/index.html

```diff
@@ -4,58 +4,68 @@
+                <div class="d-flex flex-column flex-lg-row">
+                  <div class="nav-flex-fill">
+                    <a class="dropdown-item dropdown-main-options" href="https://www.stjude.cloud/data"><img alt="data-icon" class="icon-nav" src="../../dist/images/icon-nav-data.png">Data Browser</a>
+                    <a class="dropdown-item dropdown-main-options" href="https://www.stjude.cloud/tools"><img alt="tools-icon" class="icon-nav" src="../../dist/images/icon-nav-tools.png">Analysis Tools</a>
+                    <a class="dropdown-item dropdown-main-options" href="https://www.stjude.cloud/visualizations"><img alt="visualizations-icon" class="icon-nav" src="../../dist/images/icon-nav-visualization.png">Curated Visualizations</a>
+                  </div>
+                  <div class="nav-flex-fill">
+                    <a class="dropdown-column-header dropdown-item" href="">Apps</a>
+                    <a class="dropdown-item" href="https://pecan.stjude.cloud">PeCan</a>
+                    <a class="dropdown-item" href="https://pecan.stjude.cloud">Platform</a>
+                    <a class="dropdown-column-header dropdown-item" href="">Studies</a>
+                  </div>
+                  <div class="nav-flex-fill">
+                    <p class="dropdown-column-header">Help</p>
+                    <a class="dropdown-item" href="https://www.stjude.cloud/docs/" target="_blank">User Guides</a>
+                    <a class="dropdown-item" href="https://www.stjude.cloud/faq" target="_blank">FAQ</a>
+                    <a class="dropdown-item" href="https://www.stjude.cloud/contact" target="_blank">Contact Us</a>
+                  </div>
+                </div>
+             </div>
             </li>
+            
             <li class="nav-item nav-item-user dropdown">
-              <a class="nav-link dropdown-toggle" href="/" id="nav-item-icon-user" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" title="User Info">
+              <a class="nav-link dropdown-toggle " href="/" id="nav-item-icon-user" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" title="User Info">
                 <i class="fa fa-user"></i>
               </a>
-              <div class="dropdown-menu" aria-labelledby="nav-item-icon-user">
-                <a class="dropdown-item" href="/oauth_login">Sign In</a>
-                <a class="dropdown-item" href="/user/details">Create Account</a>
+              <div class="dropdown-menu dropdown-menu-right dropdown-menu-profile" aria-labelledby="nav-item-icon-user" id="dropdown-user">
+                <div class="nav-flex-fill">
+                  <a class="dropdown-item profile-menu-dropdown-item" href="/oauth_login">Sign In</a>
+                  <a class="dropdown-item profile-menu-dropdown-item" href="/user/details">Create Account</a>
```

Comment:
Same as above.
This doesn't need an actual location, is it's only for demo purposes.
```suggestion
                  <a class="dropdown-item profile-menu-dropdown-item" href="">Create Account</a>
```